lte/gateway/c/session_manager/ShardTracker.cpp

```diff
@@ -53,4 +56,20 @@ bool ShardTracker::remove_ue(const std::string imsi, const uint16_t shard_id) {
   return true;
 }
 
+uint16_t ShardTracker::get_shard_list_size() {
+  return imsis_per_shard_.size();
+}
+
+std::vector<int> ShardTracker::get_active_shards() {
```

Comment:
int data type because cookie values passed in must be ints(but the number of shards itself will probably be small enough to be uint16).